ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -194,6 +197,22 @@ void compact(CompactionDetails details, byte[] bundleReadBuffer) throws IOExcept
     resumeCompaction(bundleReadBuffer);
   }
 
+  /**
+   * Converts the set of {@link Container}s to the set of accountId & ContainerId pairs for easy detection.
+   */
+  private void getAllFilteredContainers() {
```

Comment:
Minor comments about comment and naming. 

1. Comment its main functionality first and then describe what does it do. 
2. There is actually not `Converts` here. It's filter. 
3.  getAllFilteredContainers() -> getNotActiveContainers() and comment what is not active: DELETE_IN_PROGRESS and INACTIVE

Reply:
Updated.